mysql-compatibility.md

```diff
@@ -93,171 +76,118 @@ mysql> select _tidb_rowid, id from t;
-        - The default SQL mode in MySQL 8.0 includes these modes: `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION`.
+        - The default SQL mode in MySQL 8.0 includes these modes: ``ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION`.
 - Default value of `lower_case_table_names`:
-    - The default value in TiDB is 2 and currently TiDB only supports 2.
+    - The default value in TiDB is `2` and currently TiDB only supports `2`.
     - The default value in MySQL:
-        - On Linux: 0
-        - On Windows: 1
-        - On macOS: 2
+        - On Linux: `0`
+        - On Windows: `1`
+        - On macOS: `2`
 - Default value of `explicit_defaults_for_timestamp`:
     - The default value in TiDB is `ON` and currently TiDB only supports `ON`.
     - The default value in MySQL:
-        - For MySQL 5.7: `OFF`
-        - For MySQL 8.0: `ON`
+        - For MySQL 5.7: `OFF`.
+        - For MySQL 8.0: `ON`.
 
 ### Date and Time
 
 #### Named timezone
 
-TiDB supports named timezones such as `America/Los_Angeles` without having to load the [time zone information tables](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation) as in MySQL.
-
-Because they are built-in, named time zones in TiDB might behave slightly differently to MySQL, and cannot be modified. For example, in TiDB the names are case-sensitive [#8087](https://github.com/pingcap/tidb/issues/8087).
++ TiDB uses all timezone rules currently installed in the system for calculation (usually the tzdata package). You can use all timezone names without importing the timezone table data. You cannot modify the calculation rules by importing the timezone table data.
++ MySQL uses the local timezone by default and relies on the current timezone rules built into the system (such as when to start daylight saving time) for calculation; and the timezone cannot be specified by the timezone name without [importing the timezone table data](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation).
 
 > **Note:**
 >
-> TiKV calculates time-related expressions that can be pushed down to it. This calculation uses the built-in time zone rule and does not depend on the time zone rule installed in the system. If the time zone rule installed in the system does not match the version of the built-in time zone rule in TiKV, the time data that can be inserted might result in a statement error in a few cases.
+> TiKV calculates time-related expressions that can be pushed down to it. This calculation uses the built-in timezone rule and does not depend on the timezone rule installed in the system. If the timezone rule installed in the system does not match the version of the built-in timezone rule in TiKV, the time data that can be inserted might result in a statement error in a few cases.
```

Comment:
Why change `time zone` to `timezone`? Better keep it consistent with [that in MySQL](https://dev.mysql.com/doc/refman/5.7/en/time-zone-support.html).
```suggestion
> TiKV calculates time-related expressions that can be pushed down to it. This calculation uses the built-in time zone rule and does not depend on the time zone rule installed in the system. If the time zone rule installed in the system does not match the version of the built-in time zone rule in TiKV, the time data that can be inserted might result in a statement error in a few cases.
```